### PR TITLE
Adding  Authentication  page

### PR DESCRIPTION
**Pull request by bipin7yadav**
-Rules  page added
-Login and SignUp page added
-Auth Added

### CODE REVIEW

File: src/pages/login-signUp/auth.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    height: 75%;
+    box-sizing: border-box;
+    width: 24rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: .7rem;
+}
+.lContent{
+    /* background-color: aqua; */
```
Comment (manojkumar8400) [inline src/pages/login-signUp/auth.css:20]: remove unnecessary comment.

File: src/pages/index.js [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+import { SignUp } from "./login-signUp/signup";
+import { Rules } from "./rules/rules";
 
 
-export {LandingPage,Category}
\ No newline at end of file
+
+
+
+export {LandingPage,Category,Login,SignUp,Rules}
```
Comment (manojkumar8400) [inline src/pages/index.js:11]: You can directly export the components using barrel export.

**Review by manojkumar8400 — APPROVED** [review]
else code looks clean.

**Review by manojasarsa — APPROVED** [review]
Code looks clean. GTG!

**Pull request merged**